haystack/components/converters/docx.py

```diff
@@ -47,13 +46,13 @@ class DOCXMetadata:
     category: str
     comments: str
     content_status: str
-    created: Optional[datetime]
+    created: Optional[str]
```

Comment:
Just curious, why is it marked as optional?

Reply:
> Date properties return None if not set.

from https://python-docx.readthedocs.io/en/latest/api/document.html#coreproperties-objects